app/src/test/java/ca/llamabagel/transpo/data/DbUtils.kt

```diff
@@ -8,7 +8,7 @@ import ca.llamabagel.transpo.data.db.*
 import ca.llamabagel.transpo.di.TransitDatabaseModule.Companion.STOP_ADAPTER
 import com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver
 
-fun getDatabase(): TransitDatabase {
+fun getTransitDatabase(): TransitDatabase {
```

Comment:
```suggestion
fun provideFakeTransitDatabase(): TransitDatabase {
```

Should we use a standard naming convention for the testing dependency injection methods?